### PR TITLE
removed useless forms test,declaration,on BootstrapPage (#1326)

### PR DESCRIPTION
**Pull request by VeronikaRevjakina**
NEEDS DISCUSSION
#1326

### CODE REVIEW

**Comment by VeronikaRevjakina** [comment]
Removed:

    all tests with obsolette Forms
    +classes for its declarations (e.g.jdi-light-bootstrap-tests\src\main\java\io\github\com\sections\form\FormBrowserDefaults.java and so on)
    mentions on BootstrapPage
    whole (34 files)

Left 2 unused classes: (SHOULD BE DELETED OR NOT?)
entities:(\jdi-light-bootstrap-tests\src\main\java\io\github\com\entities)
FormSignUp.java
FormUsers.java

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/SwitchesTests.java [hunk 1]
```diff
@@ -1,85 +0,0 @@
-package io.github.epam.bootstrap.tests.composite.section.form;
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/SwitchesTests.java]: Please, restore this test and it's PO

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/form/Switches.java [hunk 2]
```diff
@@ -1,14 +0,0 @@
-package io.github.com.sections.form;
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/form/Switches.java]: Please, restore this PO

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java [hunk 3]
```diff
@@ -749,10 +708,4 @@
     public static ModalOptionalSizes modalOptionalSizes;
     //endregion
 
-    //region Forms
-    @UI("#form-horizontal-label-size")
-    public static FormHorizontalLabelSizing formHorizontalLabelSizing;
-    @UI("#form-column-size")
-    public static FormColumnSizing formColumnSizing;
-    //endregion
 }
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: You should leave an empty line at the end of file

Comment (VeronikaRevjakina) [reply jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java]: There is an empty line

**Pull request merged**